src/utp_stream.cpp

```diff
@@ -651,10 +651,7 @@ void utp_socket_impl::issue_write()
 	m_written = 0;
 	if (test_socket_state()) return;
 
-	// try to write. send_pkt returns false if there's
-	// no more payload to send or if the congestion window
-	// is full and we can't send more packets right now
-	while (send_pkt());
+	write_some(false);
```

Comment:
if you revert this, then `write_some()` wouldn't need to take the `bool` parameter, right?
it's not clear that reusing this function here is worth it. it's a single line of code, and a `bool` to disable two other lines of code in there.

Reply:
Sure, actually I only added it so it is symmetrical with `read_some`.